swift/probe/README.md

```diff
@@ -1,15 +1,45 @@
 # Moonlight
 Probe written in [swift](https://www.swift.org/)
 
-## Getting started
+## Mac Getting started
 
-Compile Moonlight, replacing ```<TARGET>``` with a supported operating system:
+Assuming you are on an ARM mac.
```

Comment:
this README needs to be clearer on the steps to install this probe on each system: Linux and MacOS right now. 

we should organize it as:

- what is this probe: description
- how to download a precompiled probe via our API
- how to install this probe from source from a Mac ARM, Mac intel, Linux intel (separate sections for each)

Reply:
Completely fair. Updated